fetch.bs

```diff
@@ -4058,6 +4511,22 @@ Range Requests</cite>. [[HTTP-RANGE]] However, this is not widely supported by b
    <li><p>If <var>connection</var> is failure, return a
    <a>network error</a>.
 
+   <li><p>If <var>request</var>'s <a for=request>use-token-binding flag</a>
+   is set, build and add a `<a http-header><code>Sec-Token-Binding</code></a>`
+   request <a for=/>header</a> by running the following substeps:
+
+    <ol>
+     <li><p>Let <var>tokenBindingMessage</var> be the result of
+     <a abstract-op lt="Compute a Token Binding header value">computing a Token Binding header value</a>
+     for <var>request</var> and <var>connection</var>,
+     using the user agent's <a for=/>token-binding key store</a> and
```

Comment:
I do not think we need the phrase beginning with "using" because that info is baked into the definition of the "Compute a Token Binding header value" alg